SimConnectMSFS/WasmModuleUpdater.cs

```diff
@@ -140,7 +140,6 @@ public bool InstallWasmModule(string communityFolder)
             // Remove the old Wasm File
             DeleteOldWasmFile(communityFolder);
 
-            Log.Instance.log("WASM module has been installed successfully.", LogSeverity.Info);
```

Comment:
The caller now logs this based on the return value so different log messages can be sent based on the simulator version.